tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -25,6 +25,9 @@
 import io.dockstore.openapi.client.api.Ga4Ghv20Api;
 import io.dockstore.openapi.client.model.Execution;
 import io.dockstore.openapi.client.model.FileWrapper;
+import io.dockstore.openapi.client.model.WorkflowSubClass;
+import io.swagger.client.api.WorkflowsApi;
```

Comment:
this is mixing swagger and openapi (above) classes. 
See if you can use openapi only

Reply:
Fixed. We only use openapi imports. This caused a lot of changes.

I think it may've broke a small bit of code. However, I don't believe we were using that code anyway.